app/analytics.go

```diff
@@ -244,7 +244,7 @@ func (a *App) GetAnalytics(name string, teamId string) (model.AnalyticsRows, *mo
 		commandChan := make(chan store.StoreResult, 1)
 		go func() {
 			c, err := a.Srv().Store.Command().AnalyticsCommandCount(teamId)
-			commandChan <- store.StoreResult{Data: c, Err: err}
+			commandChan <- store.StoreResult{Data: c, NErr: err}
```

Comment:
We have missed actually checking this error below. In L310, we should be checking for NErr rather than Err.